shell/server/core.js

```diff
@@ -320,6 +316,7 @@ Meteor.methods({
   },
 });
 
+// TODO(now): Get rid of this maybe?
```

Comment:
What do you think the best approach would be for removing `saveFrontendRef()`? Replace all uses with `globalFrontendRegistry.()` followed by a `save()`?

Looks like there are only three uses of `saveFrontendRef()`, two of which are left over from the old admin interface and are no longer accessible.